coordinator/src/routes/tracker/task.ts

```diff
@@ -0,0 +1,104 @@
+import Hapi from "@hapi/hapi"
+import {fhir} from "@models"
+import {BASE_PATH, ContentTypes} from "../util"
+import {getStatusCode} from "../../utils/status-code"
+
+const validQueryParams = ["identifier", "focus:identifier"]
+
+const sandboxSuccessResponse = (prescriptionId: string): Partial<fhir.Task> => {
+  const medicationDescription = fhir.createCodeableConcept(
+    "http://snomed.info/sct",
+    "16076005",
+    "Prescription"
+  )
+  const medicationIdentifier = fhir.createIdentifierReference(fhir.createIdentifier(
+    "https://tools.ietf.org/html/rfc4122",
+    "ee86a018-779c-4809-999f-a9d89cdfd30f"
+  ))
+  const medicationStatus: Array<fhir.ExtensionExtension<fhir.CodeableConceptExtension>> = [{
+    url: "https://fhir.nhs.uk/StructureDefinition/Extension-EPS-DispensingInformation",
+    extension: [{
+      url: "dispenseStatus",
```

Comment:
Can't see this extension in the spec: https://simplifier.net/ukdigitalmedicine/extensionrepeatdispensinginformation

Maybe Kevin hasn't added it yet?

Reply:
seems like this is the correct extension https://simplifier.net/nhsdigitalspine/extensionepsdispensinginformation

Reply:
Ah I see, will ask Kevin to fix